src/api-reference/budget/budget-trackingfield.md

```diff
@@ -1,122 +1,159 @@
-| `displayName`                 | `string`   | -      | **Required** The user-facing name                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| `hierarchyCode`               | `string`   | -      | The key value that we'll use to pass in this field value in budget items and spending items. If this is not supplied, it defaults to the next available integer.                                                                                                                                                                                                                                                                                                       |
-| `lastModifiedDate`            | `datetime` | -      | The last time this budget tracking definition was updated. Date is in GMT **READ ONLY**                                                                                                                                                                                                                                                                                                                                                                                |
-| `status`                      | `string`   | -      | **Required** The status of this budget tracking field. Valid options are OPEN and REMOVED.                                                                                                                                                                                                                                                                                                                                                                             |
-| `syncGuid`                    | `string`   | -      | The budget service's key for this object.                                                                                                                                                                                                                                                                                                                                                                                                                              |
+### <a name="budgetTracking"></a>Budget Tracking Field
+
+Name | Type | Format | Description
+-----|------|--------|------------
```

Comment:
Remove the extraneous whitespace.

Reply:
...and use the simplified table header found in the documentation template.

Reply:
:+1: